CheckboxesPage/CheckboxesPage.cs

```diff
@@ -0,0 +1,36 @@
+﻿using OpenQA.Selenium;
+
+namespace Checkboxes
+{
+    public class CheckboxesPage
+    {
+        private readonly IWebDriver _driver;
+        private readonly By allCheckboxes = By.XPath("//*[@type='checkbox']");
+        private readonly By firstCheckbox = By.XPath("//*[@id='checkboxes']/input[1]");
+
+        public CheckboxesPage(IWebDriver driver)
+        {
+            _driver = driver;
+        }
+
+        public int CountCkeckboxes()
```

Comment:
it can be lambda function `public int CountCkeckboxes() => _driver.FindElements(allCheckboxes).Count();`

Reply:
Check everywhere

Reply:
Understood, changed